definitions/windows.yaml

```diff
@@ -103,6 +103,19 @@ sources:
 supported_os: [Windows]
 urls: ['http://dfrws.org/2015/proceedings/presentations/DFRWS2015-pres3.pdf']
 ---
+name: ControlPanelFilePaths
+doc: Control Panel item registration.
```

Comment:
It would be good to give the user a bit more of a clue about why these are all interesting. e.g. "DLLs listed here will be run when the user opens the Control Panel" or similar.